FlashDevelop/MainForm.cs

```diff
@@ -192,17 +194,28 @@ public ToolStrip ToolStrip
         public ToolStripPanel ToolStripPanel
         {
             get { return this.toolStripPanel; }
-        }
-
+        }
+
         /// <summary>
-        /// Gets the toolStripStatusLabel
+        /// Gets the toolStripStatusLabel.
+        /// Use <see cref="StatusLabelText"/> instead to modify the status text.
         /// </summary>
         public ToolStripStatusLabel StatusLabel
         {
             get { return this.toolStripStatusLabel; }
         }
 
         /// <summary>
+        /// Sets the text of the <see cref="StatusLabel"/>.
+        /// Use this method instead of directly accessing the <code>Text</code> property of <see cref="StatusLabel"/>.
+        /// </summary>
+        public string StatusLabelText
+        {
+            get { return this.StatusLabel.Text; }
+            set { if (!lockStatusLabel) StatusLabel.Text = value ?? " "; }
```

Comment:
@elsassph https://github.com/fdorg/flashdevelop/pull/1245#issuecomment-225375273
>when starting a shortcut (eg. Ctrl+E), I see in the status bar that FD is waiting for another key; but if I click in the editor (to cancel the sequence), I see a strange relayout whereby the Scintilla editor grows over the bottom tabs and quickly goes back to normal size.

Apparently in HDPI (I assume) having 0-length text (`null` gets converted to `""`) in status label resizes the control.
`value ?? " "` fixes the issue, while still allowing the option to set 0-length text that resizes the control.

An alternative is just `string.IsNullOrEmpty(value) ? " " : value`.